test/unit/LoopbackModelBase.test.js

```diff
@@ -0,0 +1,48 @@
+const { expect } = require('chai');
+
+const LoopbackModelBase = require('../../src/LoopbackModelBase');
+
+const MockApp = require('../support/MockApp');
+
+describe('The LoopbackModelBase Class', function() {
+    it('throws an error if no valid loopback model is given', function(){
+        try {
+            new LoopbackModelBase({ });
+        } catch (error) {
+            expect(error).to.have.property('message', 'No Loopback Model Instance was given to the\n                LoopbackModelBase constructor');
+        }
+    });
+
+    it('creates an instance with the given values', function(){
+        const instance = new LoopbackModelBase({ model: {}, modelName: 'test' });
+
+        expect(instance).to.have.property('modelName', 'test');
+    });
+
+    it('throws an error when calling get env without loopback booted', function(){
+        const instance = new LoopbackModelBase({ model: {}, modelName: 'test' });
+
+        try {
+            instance.getEnv();
```

Comment:
This test is meaningless since it will not fail if the `getEnv` method does not throw an error.